### PR TITLE
Nf 643 review amend

### PR DESCRIPTION
**Pull request by rorticus0502**

### CODE REVIEW

File: test/controllers/AmendConfirmationControllerSpec.scala [hunk 1]
```diff
@@ -44,8 +45,10 @@ class AmendConfirmationControllerSpec extends SpecBase {
 
         status(result) mustEqual OK
 
+        val checkYourAnswersHelper = new CheckYourAnswersHelper(answers)
+
         contentAsString(result) mustEqual
-          view(claimId)(request, messages).toString
+          view(claimId, checkYourAnswersHelper.getAmendCheckYourAnswerSections)(request, messages).toString
```
Comment (njmoorewitchcat) [inline test/controllers/AmendConfirmationControllerSpec.scala:51]: RED: I was expecting to see a set of tests here for "AmendConfirmation Controller for summary view" in the same way that ConfirmationControllerSpec has

File: app/views/AmendConfirmationView.scala.html [hunk 2]
```diff
@@ -23,32 +24,46 @@
...
+        @html.p{ @messages("amend.confirmation.amendWhatHappensNext.p2") }
 
-    @html.h2("amend.confirmation.helpUs.heading", classes = "govuk-heading-m")
-    @html.p{ @messages("amend.confirmation.helpUs.p1") }
-    @html.p{
-        @html.a(config.feedbackSurvey, messages("amend.confirmation.helpUs.p2"))
-        @messages("amend.confirmation.helpUs.p3")
-    }
+        @html.p{
+            @html.a(config.feedbackSurvey, messages("confirmation.improveService.link.text1"))
```
Comment (njmoorewitchcat) [inline app/views/AmendConfirmationView.scala.html:62]: RED: Wrong message key - should be "amend.confirmation.improveService.link.text1", as we seem to be keeping the pages message keys separate.

File: app/views/AmendConfirmationView.scala.html [hunk 3]
```diff
@@ -23,32 +24,46 @@
...
+        }
+    </div>
+    <div class="ndrc-!-display-only-print">
+        @html.h1("confirmation.summary.title")Con
```
Comment (njmoorewitchcat) [inline app/views/AmendConfirmationView.scala.html:66]: RED: Wrong message key

**Review by njmoorewitchcat — CHANGES_REQUESTED** [review]

File: app/views/AmendConfirmationView.scala.html (hunk 3 above)
Comment (SteveSugden) [inline app/views/AmendConfirmationView.scala.html:66]: Remove the "Con" above

**Review by SteveSugden — CHANGES_REQUESTED** [review]
Just needs that "Con" text removing - otherwise LGTM

File: app/views/AmendConfirmationView.scala.html (hunk 3 above)
Comment (njmoorewitchcat) [reply app/views/AmendConfirmationView.scala.html:66]: > Remove the "Con" above

How did I not see that :/

File: app/views/AmendConfirmationView.scala.html (hunk 2 above)
Comment (SteveSugden) [reply app/views/AmendConfirmationView.scala.html:62]: Not sure that's a hard and fast rule.  We have examples of message keys being shared between views / journeys as well.

Comment (rorticus0502) [reply app/views/AmendConfirmationView.scala.html:62]: I considered seperating it, as it is in most places, but on balance I decided that for this particular case it was not necessary.

File: test/controllers/AmendConfirmationControllerSpec.scala (hunk 1 above)
Comment (rorticus0502) [reply test/controllers/AmendConfirmationControllerSpec.scala:51]: Tests for summary view added

File: app/views/AmendConfirmationView.scala.html (hunk 3 above)
Comment (rorticus0502) [reply app/views/AmendConfirmationView.scala.html]: Con removed

File: app/views/AmendConfirmationView.scala.html [hunk 4]
```diff
@@ -23,32 +24,46 @@
...
-    @html.p{
-        @html.a(config.feedbackSurvey, messages("amend.confirmation.helpUs.p2"))
-        @messages("amend.confirmation.helpUs.p3")
-    }
+        @html.p{
+            @html.a(config.feedbackSurvey, messages("confirmation.improveService.link.text1"))
+        }
+    </div>
+    <div class="ndrc-!-display-only-print">
+        @html.h1("confirmation.summary.title")
```
Comment (njmoorewitchcat) [inline app/views/AmendConfirmationView.scala.html:66]: RED: This is still the wrong key.  Currently the message says: "Your application details"
We are not making an application.  We are making a new application.

**Review by njmoorewitchcat — CHANGES_REQUESTED** [review]

File: app/views/AmendConfirmationView.scala.html (hunk 4 above)
Comment (rorticus0502) [reply app/views/AmendConfirmationView.scala.html:66]: Yeah but I dont know what it should say o_0

**Review by SteveSugden — DISMISSED** [review]
LGTM

**Review by njmoorewitchcat — APPROVED** [review]

**Review by SteveSugden — DISMISSED** [review]

**Review by SteveSugden — APPROVED** [review]

**Pull request merged**